content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+These are known as **"networks"**.\
+Proxies are also incredibly lightweight, with small CPU and RAM usage (typically less than 1-2GB of RAM) however this can vary depending on your player count and proxy plugins.
+
+## Velocity (recommended)
+[Velocity](https://papermc.io/software/velocity) is a modern, high-performance, and extremely flexible Minecraft proxy, maintained by the [PaperMC Team](https://papermc.io/).
+It is relatively new, but has quickly become the most popular proxy server software due to its performance and flexibility.
+
+<Callout type="info" title="Note">
+    Velocity makes use of its own API, meaning all Bungeecord plugins do **not** function on Velocity, although there are workaround plugins that introduce some degree of compatibility between the two plugin platforms.\
+    One example of this is [Snap](https://hangar.papermc.io/Phoenix616/Snap).<br></br>
+
+    Ultimately Velocity is a far more modern and superior proxy software compared to Waterfall and Bungeecord, unless you make use of plugins that only have Bungeecord support.
+</Callout>
+
+<Card href="https://jar.setup.md/download/velocity/latest/latest" title="Download">
+  Download a copy of the latest Velocity proxy software via [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## Velocity-CTD
+[Velocity-CTD](https://github.com/GemstoneGG/Velocity-CTD) is a fork of [Velocity](#velocity-recommended) that adds additional features and optimizations to the software.\
+It is fully open source and adds Quality of Life commands and more support for backend server softwares - particularly modded servers which are notoriously difficult when within a network.
+
+Installing Velocity-CTD is as simple as replacing the normal Velocity jarfile with the new Velocity-CTD one.
+
+<Card href="https://github.com/GemstoneGG/Velocity-CTD/releases/latest" title="Download">
+  Download a copy of the latest Velocity-CTD proxy software via [github.com](https://github.com/).
+</Card>
+
+## Bungeecord
+[Bungeecord](https://www.spigotmc.org/wiki/bungeecord/) is a "legacy" proxy server software that is still widely used today.\
+Like all other proxy softwares, it allows individual backend servers to be connected together allowing for easy transfer between servers.\
+However, Bungeecord is not as performant as Velocity and their API is not as extensive as Velocity's, although many plugin developers still support it.
+
+<Card href="https://ci.md-5.net/job/BungeeCord/" title="Download">
+  Download a copy of the latest Bungeecord proxy software via [ci.md-5.net](https://ci.md-5.net/).
+</Card>
+
+## Waterfall
+[Waterfall](https://papermc.io/software/waterfall) is a fork of Bungeecord that aims to improve performance and stability, focusing on Stability, Features and Scalability.
+
+<Callout type="warn" title="Warning">
+  The PaperMC development team have announced that Waterfall is no longer maintained and is EOL.\
```

Comment:
Also:

> The PaperMC development team have announced

The PaperMC development team **has** announced

"Team" refers to many people, but is a singular noun, as it is one single team. Therefore, it gets singular verbs.

Reply:
Resolved in [81ef8d1](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/81ef8d1899cd10380ab6d2f1ebf25e37a8b969a5)